rhba-dev-env/vagrant/provisioning/roles/setup_env/templates/cekit.j2

```diff
@@ -0,0 +1,11 @@
+[common]
+
+ssl_verify = {{ ssl_verify_value }}
+redhat =  {{ redhat_value }}
+
+[repositories]
+
+jboss-os = {{ jboss_os_repo }}
```

Comment:
those repos are not supported by cekit anymore, thus can be removed.

Reply:
Yup; cekit will complain.